docs/guides/platform/migrate-to-linode/migrate-from-aws-s3-to-linode-object-storage/index.md

```diff
@@ -0,0 +1,308 @@
+
+-   `--log-file=rclone.log`: The file where rclone writes logs. This file is created in the working directory from where the `copy` command is run.
+
+-   `--log-level=ERROR`: The type of logs to be written to your log file. `ERROR` here specifies only errors are written to the `rclone.log` file.
+
+-   `--rc-web-gui`: Serves the WebUI GUI on the default rclone port (5572).
+
+-    `--rc-user {{< placeholder "USERNAME" >}} and --rc-pass {{< placeholder "PASSWORD" >}}`: The username and password used to access the WebUI GUI.
+
+{{< note title="Using the htpasswd flag" >}}
+An alternative to the `--rc-user` and `--rc-pass` combination is the `--rc-htpasswd` flag. This creates a `htpasswd` file containing a generated username and password combination you can use to log into the rclone WebUI. See [Remote controlling rclone with its API](https://rclone.org/rc/#rc-htpasswd-path)
+{{< /note >}}
+
+-   `--bind ::0` (for use with Architecture 2): Tells rclone to write data via IPv6. Note that writing data over IPv6 from a Linode instance to an Object Storage bucket in the same region is free of charge.
+
+#### Optional Flags
+
+-   `--tpslimit {{< placeholder "XXX" >}}` : Specifies the number of HTTP transactions per second. For larger transfers, it is considered a best practice to set the `tpslimit` below the infrastructure requests per second (rps) limit. Should an rps limit be reached, a 503 `SlowDown` error may result.
+
+    **Example:** If the infrastructure’s requests per second limit is 750 rps, set the tpslimit to 725:
+
+    ```command
+    --tpslimit {{< placeholder "725" >}}
+    ```
+
+### Monitoring the Migration
+
+To monitor the status of the `rclone copy` command above, you can access the rclone WebUI GUI from a web browser.
+
+1.  In a web browser window, navigate to your instance’s address over port 5572. Replace {{< placeholder "IP-ADDRESS" >}} with the IPv4 address of your instance:
+
+    ```command
+    http://{{< placeholder "IP-ADDRESS" >}}:5572
+    ```
+
+1.  When prompted, enter the username and password you specified with the `--rc-user` and `--rc-pass` flags:
+
+    ![Rclone-WebUI-Login](Rclone-WebUI-Login.jpg)
+
+1.  Once logged in, you should see active running jobs along with multiple monitoring statistics, including: job status, throughput and speed, bandwidth max speed, total objects transferred, amount of data transferred, and more.
+
+    ![Rclone-global-stats](Rclone-global-stats.jpg)
+
+    ![Rclone-speed](Rclone-speed.jpg)
+
+1.  The WebUI will disconnect automatically when the copy job is complete.
+
+## Verify the Migration
+
+You can compare the number of objects in both your source and target buckets along with the total size of the buckets to verify full completion of the copy job.
```

Comment:
I'm wondering if there's some kind of checksum/hash comparison that could be done that also verifies the contents of the buckets. There might not be, but it was just a thought that came to me

Reply:
will check on this with my SME for a future edit